packages/common/http/src/context.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * A token used to manipulate and access values stored in HttpContext
```

Comment:
Type references are usually backticked.

```suggestion
 * A token used to manipulate and access values stored in `HttpContext`.
```

There are some more occurrences like these. Also notice the period at the end of the sentence :)